.github/workflows/ci.yaml

```diff
@@ -2,6 +2,8 @@ name: Ansible Docsite CI
 
 on:
   push:
+    branches-ignore:
+      - 'patchback/**'
```

Comment:
@gotmax23 FYI I also tend to ignore pre-commit and dependabot branches: https://github.com/ansible/pylibssh/blob/b9581d2/.github/workflows/ci-cd.yml#L10-L13 / https://github.com/cherrypy/cheroot/blob/00c3cc3/.github/workflows/ci-cd.yml#L6-L11

Reply:
We can add those when/if we adopt those tools. I don't want them there just cause.